internal/models/access_flow.go

```diff
@@ -0,0 +1,61 @@
+package models
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// AccessFlowModel describes the resource data model.
+type AccessFlowModel struct {
+	ID                 types.String        `tfsdk:"id"`
+	Name               types.String        `tfsdk:"name"`
+	Active             types.Bool          `tfsdk:"active"`
+	RevokeAfterInSec   types.Number        `tfsdk:"revoke_after_in_sec"`
+	Trigger            *Trigger            `tfsdk:"trigger"`
+	Grantees           types.Set           `tfsdk:"grantees"`
+	IntegrationTargets []IntegrationTarget `tfsdk:"integration_targets"`
+	Approvers          types.Set           `tfsdk:"approvers"`
```

Comment:
why grantees/approvers are types.Set but targets is a []InectrationTarget list with properly defined type?